tests/CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ set(TESTS
     test_algorithm
     test_raster
     test_triangle
+    test_triangle_subdivision
```

Comment:
Ich würde sagen, dass wir den test ruhig in test_triangle reinpacken können -> weniger kompilieren.